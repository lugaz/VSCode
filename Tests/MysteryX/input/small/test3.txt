getRear
insertRear 288
deleteFront
getFront
insertRear 135
getFront
getFront
insertRear 137
getFront
deleteFront
insertFront 867
insertFront 206
deleteRear
getRear
getRear
deleteRear
getFront
getFront
insertRear 441
insertFront 922
getRear
deleteRear
deleteRear
deleteFront
getFront
deleteRear
insertFront 10
deleteRear
getRear
getRear
deleteRear
insertRear 510
getRear
insertFront 859
