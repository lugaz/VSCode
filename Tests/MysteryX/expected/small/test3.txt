-1
288
-1
135
135
135
135
137
867
867
867
206
206
441
441
206
922
-1
-1
10
-1
-1
-1
510
